examples/socketio/README.md

```diff
@@ -0,0 +1,79 @@
+# socketio
```

Comment:
Let's add a "Stability: ⚠️Experimental⚠️" heading like in https://github.com/strongloop/loopback-next/blob/master/extensions/typeorm/README.md and others.

This will allow us to make changes without the burden of backwards compatibility issues. After a certain amount of time and usage, once the extension has matured, we can remove it.